code/index.html

```diff
@@ -3,28 +3,198 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Pawsitive talk</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet">
+    <link rel="icon" type="image/png" href="./dog-paw.png"/>
   </head>
+
   <body>
+    <header>
+      <nav>
+        <a href="#">Who we are</a>
+        <a href="#">Our mission</a>
+        <a href="#">
+          <img class="nav-image" src="./logo_transparent.png" alt="Pawsitive talk logo"/>
+        </a>
+        <a href="#">Our methods</a>
+        <a href="#">Contact</a>
+      </nav>
+      <video autoplay loop muted>
+        <source src="./videos/dog-video.mp4" type="video/mp4">
+        <p>This HTML5 video is not supported by the browser you're using.</p>  
+      </video>
     <!-- video or image as a header is cool :) -->
+    </header>
 
-    <!-- Signup form -->
+    <main>
+      <!-- Signup form -->
+      <section class="form">
+        <form action="http://httpbin.org/anything" method="POST">
+          <div class="form-title">
+            There is a solution to every behavioural issue. Trust us. 
+            We're dog behaviourists.
+            Get in touch for a consultation session!
+          </div>
+          
+          <!--OWNER'S INFO-->
+          <fieldset class="owner-info">
+          <legend>Information about owner</legend>
+          <div class="owner-info-div">
+            <label>
+              Your full name:
+              <input type="text" required name="fullName" class="input-owner">
```

Comment:
Nice that you've used required, however all of your input tags need to be closed. Either self-closing or with a `</input>`